nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -193,6 +193,10 @@
 
 - The `stackclashprotection` hardening flag has been added, though disabled by default.
 
+- `buildPythonPackage` and `buildPythonApplication` now take `doCheck`-related attributes (`doInstallCheck`, `installCheckPhase`, `installCheckInputs`, `nativeInstallCheckInputs`) directly, which precedes the old approach of specifying them via the `check`-related arguments (`doCheck`, `checkPhase`, `checkInputs`, `nativeCheckInputs`).
+
+    Existing Python packages and applications will stay unchanged as long as they don't specify `doCheck` and `doInstallCheck` differently. Such specification is irrelevent both before and after this change, but the effect would be different.
```

Comment:
```suggestion
- The `buildPythonPackage` and `buildPythonApplication` functions now consume the `installCheckPhase` attributes (`doInstallCheck`, `installCheckPhase`, `installCheckInputs`, `nativeInstallCheckInputs`) directly. `checkPhase` attributes (`doCheck`, `checkPhase`, `checkInputs`, `nativeCheckInputs`) have been aliases to their `installCheckPhase` counterparts since at least 2016 and their usage is now deprecated.
```